File: datasets/business/openshop/modify/fragment_account.swift
import SwiftUI
struct fragment_account: View {

var body: some View {
ZStack
{
VStack
{
VStack
{
Text("Name:")
.font(.system(size: 12))
.foregroundColor(Color(red:0.44705883,green:0.44705883,blue:0.44705883,opacity:1.0))
Text("User and Surname")
.font(.system(size: 16))
.foregroundColor(Color(red:0.12941177,green:0.12941177,blue:0.12941177,opacity:1.0))
Text("Address:")
.padding(.top,16)
.font(.system(size: 12))
.foregroundColor(Color(red:0.44705883,green:0.44705883,blue:0.44705883,opacity:1.0))
Text("Brnenska 14, Brno, 61200")
.padding(.trailing,6)
.padding(.trailing,6)
.font(.system(size: 16))
.foregroundColor(Color(red:0.12941177,green:0.12941177,blue:0.12941177,opacity:1.0))
Text("Email:")
.padding(.top,16)
.font(.system(size: 12))
.foregroundColor(Color(red:0.44705883,green:0.44705883,blue:0.44705883,opacity:1.0))
Text("test@mailing.com")
.font(.system(size: 16))
.foregroundColor(Color(red:0.12941177,green:0.12941177,blue:0.12941177,opacity:1.0))
Text("Phone:")
.padding(.top,16)
.font(.system(size: 12))
.foregroundColor(Color(red:0.44705883,green:0.44705883,blue:0.44705883,opacity:1.0))
Text("903578123")
.font(.system(size: 16))
.foregroundColor(Color(red:0.12941177,green:0.12941177,blue:0.12941177,opacity:1.0))
}
.padding(.trailing,200)
VStack
{
HStack
{
Button(action: {})
{
Text("Account")
.foregroundColor(.black)
}
.frame(minWidth:0,maxWidth:.infinity)
.background(Color.gray)
ZStack()
{}
.frame(minWidth:16,idealWidth:16)
Button(action: {})
{
Text("My orders")
.foregroundColor(.black)
}
.frame(minWidth:0,maxWidth:.infinity)
.background(Color.gray)
}
.padding(.top,16)
HStack
{
Button(action: {})
{
Text("Branches")
.foregroundColor(.black)
}
.frame(minWidth:0,maxWidth:.infinity)
.background(Color.gray)
ZStack()
{}
.frame(minWidth:16,idealWidth:16)
Button(action: {})
{
Text("Settings")
.foregroundColor(.black)
}
.frame(minWidth:0,maxWidth:.infinity)
.background(Color.gray)
}
.padding(.top,16)
Button(action: {})
{
Text("Log in")
        .foregroundColor(.white)
}
.background(Color.pink)
.padding(.top,16)
.frame(minWidth:0,maxWidth:.infinity)
}

.padding(.top,16)
}
.padding(16)
}}}

struct fragment_account_Previews: PreviewProvider {
    static var previews: some View {
        fragment_account()
    }
}

File: datasets_v2/business/openshop/compose/fragment_account.kt

import androidx.compose.runtime.Composable
import androidx.compose.ui.tooling.preview.Preview

import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.alpha
import androidx.compose.ui.graphics.Color
import androidx.compose.foundation.Image
import androidx.compose.foundation.rememberScrollState
import androidx.compose.foundation.verticalScroll
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.unit.sp
import androidx.compose.ui.unit.dp
@Composable
fun fragment_account(){  androidx.compose.material3.Surface(
    modifier = Modifier.fillMaxWidth(),
    shape = androidx.compose.foundation.shape.RoundedCornerShape(16.dp),
    color = androidx.compose.material3.MaterialTheme.colorScheme.surface
)  {
    Column(modifier = Modifier.wrapContentHeight().verticalScroll(rememberScrollState())) {
        Column(modifier = Modifier.fillMaxWidth().wrapContentHeight().padding(16.0.dp)) {
            Column(modifier = Modifier.wrapContentHeight()) {
                Text(text = "Name:", modifier = Modifier.wrapContentWidth().wrapContentHeight(), color = Color(0xFF727272), fontSize = 12.sp)
                Text(text = "User and Surname", modifier = Modifier.wrapContentWidth().wrapContentHeight(), color = Color(0xFF212121), fontSize = 16.sp)
                Text(text = "Address:", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(top = 16.0.dp), color = Color(0xFF727272), fontSize = 12.sp)
                Text(text = "Brnenska 14, Brno, 61200", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(end = 6.0.dp), color = Color(0xFF212121), fontSize = 16.sp)
                Text(text = "Email:", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(top = 16.0.dp), color = Color(0xFF727272), fontSize = 12.sp)
                Text(text = "test@mailing.com", modifier = Modifier.wrapContentWidth().wrapContentHeight(), color = Color(0xFF212121), fontSize = 16.sp)
                Text(text = "Phone:", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(top = 16.0.dp), color = Color(0xFF727272), fontSize = 12.sp)
                Text(text = "903578123", modifier = Modifier.wrapContentWidth().wrapContentHeight(), color = Color(0xFF212121), fontSize = 16.sp)
            }
            Column(modifier = Modifier.fillMaxWidth().wrapContentHeight().padding(top = 16.0.dp)) {
                Row(modifier = Modifier.fillMaxWidth().wrapContentHeight().padding(top = 16.0.dp)) {
                    Button(onClick = {}, enabled = true, modifier = Modifier.wrapContentHeight().width(0.0.dp).weight(1.0f)) {
                        Text(text = "Account")
                    }
                    Column(modifier = Modifier.fillMaxHeight().width(16.0.dp)) {
                    }
                    Button(onClick = {}, enabled = true, modifier = Modifier.wrapContentHeight().width(0.0.dp).weight(1.0f)) {
                        Text(text = "My orders")
                    }
                }
                Row(modifier = Modifier.fillMaxWidth().wrapContentHeight().padding(top = 16.0.dp)) {
                    Button(onClick = {}, enabled = true, modifier = Modifier.wrapContentHeight().width(0.0.dp).weight(1.0f)) {
                        Text(text = "Branches")
                    }
                    Column(modifier = Modifier.fillMaxHeight().width(16.0.dp)) {
                    }
                    Button(onClick = {}, enabled = true, modifier = Modifier.wrapContentHeight().width(0.0.dp).weight(1.0f)) {
                        Text(text = "Settings")
                    }
                }
                Button(onClick = {}, enabled = true, modifier = Modifier.fillMaxWidth().wrapContentHeight().padding(top = 16.0.dp)) {
                    Text(text = "Log in")
                }
            }
        }
    }
}
}



